pkg/reconciler/pipelinerun/pipelinerun.go

```diff
@@ -80,6 +80,9 @@ const (
 	// ReasonCouldntGetCondition indicates that the reason for the failure status is that the
 	// associated Pipeline's Conditions couldn't all be retrieved
 	ReasonCouldntGetCondition = "CouldntGetCondition"
+	// ParameterMissing indicates that the reason for the failure status is that the
+	// associated PipelineRun didn't provide all the required parameters
+	ParameterMissing = "ParameterMissing"
```

Comment:
This is one of the reasons pipeline validation might fail with, we have been naming such const to begin with `Reason` 😲   please change `ParameterMissing` to `ReasonParameterMissing` 🙏